src/typography/src/Text.js

```diff
@@ -1,19 +1,15 @@
 import React, { forwardRef, memo } from 'react'
 import PropTypes from 'prop-types'
 import Box from 'ui-box'
-import useTextStyle from '../../theme/src/hooks/useTextStyle'
+import useStyleConfig from '../../hooks/use-style-config'
+
+const emptyObject = {}
 
 const Text = memo(
   forwardRef(function Text(props, ref) {
-    const {
-      className,
-      color = 'default',
-      fontFamily = 'ui',
```

Comment:
We probably need to support `fontFamily` still?

Reply:
These happen in the theme itself!

Reply:
We support these as props though – you can do `<Text fontFamily="mono" />` (see `Code` for instance).